Documentation/components/k8s/pcn-kubernetes.rst

```diff
@@ -318,3 +318,8 @@ Developing
 ----------
 
 Refer to :doc:`Developers <developers>`
+
+Compatibility
+----------
+
+Although being on the latest version of Kubernetes is reccomended, pcn-k8s is compatible with all versions equal or greater than 1.9.
```

Comment:
==>
Pcn-k8s is compatible with all versions equal or greater than 1.9, although we recommend the latest version of Kubernetes.

Reply:
Just updated, thank you.